src/app/bottom-bar/bottom-bar.component.scss

```diff
@@ -11,6 +11,10 @@
   background-color: mat-color($primary);
   display: flex;
   flex-direction: row;
+  position: absolute;
```

Comment:
Didn't we decide we didn't want the bottom bar to float over the other content?

Reply:
Oh, yeah. I do think we had decided that. Is this where a change would be needed to accommodate that decision?